web/client-api/src/main/java/io/deephaven/web/client/api/JsPartitionedTable.java

```diff
@@ -248,6 +259,26 @@ public int size() {
         return tables.size();
     }
 
+    /**
+     * An array of all the key columns used by the table
```

Comment:
Javadoc should be sentences, end in a `.` - I think it might be more accurate to say that these are the key columns that the tables are partitioned by, or something to that effect?